Directory.Build.props

```diff
@@ -1,14 +1,40 @@
 <Project>
 
-  <PropertyGroup Label="Targets">
+	<!-- Versioning property for builds and packages -->
+	<PropertyGroup>
+		<VersionPrefix>1.0.40</VersionPrefix>
+	</PropertyGroup>
+
+	<PropertyGroup Label="Targets">
     <TargetFrameworks>net8.0</TargetFrameworks>
   </PropertyGroup>
-
+	
   <PropertyGroup Label="Configuration">
     <LangVersion>latest</LangVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
+	<!-- Common properties for nuget packages -->
```

Comment:
Great to see the move from `nuspec` to SDK-style properties 🎉

Reply:
We should have it out there soon. Just need to figure out how to use donet pack in conjunction with Directory.Build.props, GarnetServer.csproj Garnet.host.csproj and the release yml to NOT put .pdb in the garnet-server.nupkg and Microsoft.Garnet.nupkg.  We still want a .snupkg file that contains the symbols though.

Tried:
  <ItemGroup>
    <None Update="**\*.pdb">
      <Pack>false</Pack>
    </None>
  </ItemGroup>

and:
<AllowedOutputExtensionsInPackageBuildOutputFolder>dll</AllowedOutputExtensionsInPackageBuildOutputFolder>

Scratching my head over this one.